keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -1104,7 +1274,15 @@ For each of them, fill in the following information by copying the below templat
     - Testing: Are there any tests for failure mode? If not, describe why.
 -->
 
-- TBD (complete for beta)
+- Since all failure modes will be visible from a client command, failures should
+  be easy to detect.
+- The most straightforward failures will be error messages returned from a client
+  command.
+- Another possible failure mode would be hanging (or slow responses) from a client
+  command.
+- The simplest mitigation would be to turn off the `kubectl` feature gate.
+- The most thorough mitigation would be to restart the API server with the
+  feature gate turned off.
 
 ###### What steps should be taken if SLOs are not being met to determine the problem?
```

Comment:
I think we need something here.

Reply:
I've added a couple steps here. Please let me know what you think.

Reply:
Great. When people use these steps they are often in hurry to fix a problem and benefit significantly from having as much detail as possible.  Ideally, they can copy-paste with minimal need to formulate commands. Could we add more detail here?  E.g. "Edit ... and set the ... feature gate to ... to turn of the feature in `kubectl`.

Reply:
I have added significantly more detail on the specific steps for modifying the feature gates for `kubectl` and for the cluster. Please let me know what you think.